src/ir_Bosch.cpp

```diff
@@ -8,9 +8,11 @@
 //   Brand: Bosch,  Model: RG10A(G2S)BGEF remote
 
 #include <algorithm>
-#ifndef ARDUINO
+/*#ifndef ARDUINO
 #include <string>
-#endif
+#endif*/
+#include <cstring>
+#include "ir_Bosch.h"
```

Comment:
Why are these changes needed?

Reply:
Without <cstring> I get std::memcpy(_.raw, new_code, kBosch144StateLength); the error message "std has no member memcpy". I thought <cstring> would replace <string>.
My programming skills are very limited.
My English too.

Reply:
Without `<cstring>` I get `std::memcpy(_.raw, new_code, kBosch144StateLength);` the error message "std has no member memcpy". I thought `<cstring>` would replace `<string>`.
My programming skills are very limited.
My English too.

Reply:
`<cstring>` is fine, e.g.:
https://github.com/crankyoldgit/IRremoteESP8266/blob/85d587fbbf804c25c15632849386091e712a6cae/src/ir_Samsung.cpp#L14-L17

it was the other bit I was refering to. i.e. the removal of the `#ifndef` etc.

Reply:
is back in